examples/guestbook/README.md

```diff
@@ -368,6 +368,7 @@ The service is described in the file `examples/guestbook/frontend-service.json`:
   "apiVersion": "v1beta1",
   "port": 8000,
   "containerPort": "http-server",
+  "publicIPs":["10.11.22.33"],
```

Comment:
By changing the json here, it makes it so that this block no longer matches the file that the readme text claims it is pulled from.